### DISCUSSION TITLE
RNSScreen in analytics

### DISCUSSION THREAD

**Discussion by thomasblom** [software-mansion/react-native-screens#1831] [Q&A]
I'm using `react-native-screens` in combination with `@react-native-firebase/analytics`. In every event there is a property `firebase_screen_class` that contains the current classname of the screen.
By default `react-native-screens` sets this property as `RNSScreen`, but i change the value with `analytics().logScreenView(...)`.
After a couple of events it changes back to `RNSScreen` and i'm not sure where that happens..

Does anyone know more about this value and where it comes from? It is possible that its an issue in the Firebase Analytics package, but maybe someone here knows more.

Thanks in advance!

**Comment by tboba** [comment]
Hi @thomasblom! Sorry you had to wait for response from our side 😔
Do you know which events are causing your names to come back to the default `RNSScreen`? I'm not quite familiar with `@react-native-firebase/analytics`, but we may try to come to the solution.
